forms/DropdownField.php

```diff
@@ -110,7 +110,7 @@ protected function getFieldOption($value, $title) {
 
 	/**
 	 * @param array $properties
-	 * @return HTMLText
+	 * @return string
```

Comment:
I'm going to change this to string|HTMLText since there are so many variaties of formfields that either do raw string manipulation or use renderTemplate().